core/src/main/java/org/hibernate/ogm/type/descriptor/impl/Iso8601CalendarTypeDescriptor.java

```diff
@@ -103,12 +105,19 @@ public Calendar fromString(String string) {
 
 			@Override
 			public String toString(Calendar value) {
+				clearCalendar( value );
 				return DatatypeConverter.printDateTime( value );
 			}
 		};
 
 		public abstract Calendar fromString(String string);
 
 		public abstract String toString(Calendar value);
+
+		protected void clearCalendar(Calendar value) {
```

Comment:
Because DatatypeConverter not print miliseconds if it not set.  It leads to problem with parse date by pattern.  I will corect it by using own stored procedure. Now I know how develop stored procedures for OrientDB.